Tests/FormworksTests/Validators/FWRegexValidatorTests.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  FWRegexValidatorTests.swift
+//  FormworksTests
+//
+//  Created by Edgar Sgroi on 29/10/20.
+//
+
+import XCTest
+@testable import Formworks
+
+class FWRegexValidatorTests: XCTestCase {
+    var sut: FWRegexValidator!
+
+    override func tearDown() {
+        super.tearDown()
+        sut = nil
+    }
+    
+    // MARK: Email
+    func testEmailValidationSuccess() {
+        sut = FWRegexValidator(regex: .email)
+        
+        var result = sut.validate("test@example.com")
+        XCTAssertTrue(result)
+        
+        result = sut.validate("test01@example.com")
+        XCTAssertTrue(result)
+        
+        result = sut.validate("test@example01.com")
+        XCTAssertTrue(result)
+    }
```

Comment:
@EdgSgroi I usually prefer to add custom messages to the assertions like in this example "numbered domain" or "numbered local-part" so that in case they fail so I can more easily tell what went wrong from the console but I confess I have no arguments to support that claim.